web/src/pages/guidance/published.page.tsx

```diff
@@ -74,98 +71,135 @@ export default function Published({
 
 			<GuidanceListNav />
 
-			<p>
-				Showing {products.length} products on page {currentPage} of {totalPages}{" "}
-				({totalProducts} products total)
-			</p>
-
 			<Grid gutter="loose">
 				<GridItem cols={12} md={4} lg={3}>
-					<h2>Product types</h2>
-					<ul className="list list--unstyled">
-						{productTypes.map(({ Name }) => (
-							<li key={Name}>{Name}</li>
-						))}
-					</ul>
-					<h2>Areas of interest</h2>
-					<ul className="list list--unstyled">
-						{areasOfInterest.map(({ Name }) => (
-							<li key={Name}>{Name}</li>
+					<FilterPanel heading="Filter">
+						<FilterByInput
+							name="q"
+							label="Filter by title"
+							buttonLabel="Filter by title"
+						/>
+						{results.navigators.map(({ shortName, displayName, modifiers }) => (
+							<FilterGroup
+								key={shortName}
+								heading={displayName}
+								selectedCount={
+									modifiers.filter((modifier) => modifier.active).length
+								}
+							>
+								{modifiers.map((modifier) => (
+									<FilterOption
+										key={modifier.displayName}
+										isSelected={modifier.active}
+										onChanged={() => {
+											/* what */
+										}}
+										value={modifier.displayName}
+									>
+										{modifier.displayName}
+									</FilterOption>
+									// <li key={modifier.displayName}>
+									// 	<Link href={modifier.toggleUrl.fullUrl}>
+									// 		<a>{modifier.displayName}</a>
+									// 	</Link>
+									// </li>
+								))}
+							</FilterGroup>
 						))}
-					</ul>
+					</FilterPanel>
 				</GridItem>
 				<GridItem cols={12} md={8} lg={9}>
-					<h2>Products</h2>
-					<ol className="list list--unstyled">
-						{products.map((product) => (
-							<ProductCard
-								key={product.Id}
-								product={product}
-								productTypeName={getProductTypeName(product)}
-							/>
-						))}
-					</ol>
+					<FilterSummary
+						id="filter-summary"
+						activeFilters={activeModifiers.map((modifier) => ({
+							label: modifier.displayName,
+							destination: modifier.toggleUrl.fullUrl,
+						}))}
+						sorting={[{ label: "Date" }, { label: "Title" }]}
+					>
+						Showing {results.firstResult} to {results.lastResult} of{" "}
+						{results.resultCount}
+					</FilterSummary>
+
+					<Table aria-describedby="filter-summary">
```

Comment:
Does the `aria-describedby` effectively cover the same use-case as a `caption`? Or do we need caption too? (even if it's just visually hidden)